docs/resources/resource.md

```diff
@@ -64,6 +75,15 @@ resource "twingate_resource" "resource" {
 
 - `id` (String) Autogenerated ID of the Resource, encoded in base64
 
+<a id="nestedblock--access"></a>
+### Nested Schema for `access`
+
+Optional:
+
+- `group_ids` (Set of String) List of Group IDs that must have permission to access the Resource.
```

Comment:
Hey @vmanilo sorry I made a mistake here, should be "will" instead of "must". Full sentence: "List of Group IDs that will have permission to access the Resource."

Reply:
@alexmensch what about `service_account_ids`, do I need to fix it as well?

currently it looks like this: `List of Service Account IDs that must have permission to access the Resource.`

Reply:
Ah, yes please @vmanilo! Yeah, I'm not sure what I was thinking last night, but it should be "will" instead of "must"